examples/chip-tool/templates/logging/DataModelLogger-src.zapt

```diff
@@ -162,7 +162,7 @@ CHIP_ERROR DataModelLogger::LogEvent(const chip::app::EventHeader & header, chip
 {{/first}}
             case {{asUpperCamelCase parent.name}}::Events::{{asUpperCamelCase name}}::Id:
             {
-                {{zapTypeToDecodableClusterObjectType name ns=parent.name forceNotOptional=true}} value;
+                {{asUpperCamelCase parent.name}}::Events::{{asUpperCamelCase name}}::DecodableType value;
```

Comment:
This does not look right to me.  `zapTypeToDecodableClusterObjectType` should be producing the right thing; if it's not, it should be fixed.

I'll put up a PR to do that.

Reply:
Let's revert this in future change.

Reply:
I've raised a tracking issue here: https://github.com/project-chip/connectedhomeip/issues/30666